nsv13/code/modules/jobs/nsv13_job_mail.dm

```diff
@@ -0,0 +1,59 @@
+//ATC
+/datum/job/air_traffic_controller
+	mail_goodies = list(
+		/obj/item/pizzabox/infinite = 20,
+		/obj/item/toy/cards/deck = 15,
+		/obj/item/clothing/glasses/sunglasses/advanced/gar = 10,
+		//Since everyone wants the ATC to kys, they can have a fake one while they consider therapy (or maybe just a job change)
+		/obj/item/toy/katana = 8,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/torp_juice = 8,
+	)
+
+//Bridge Officer
+/datum/job/bridge
+	mail_goodies = list(
+		/obj/item/pizzabox/infinite = 20,
+		/obj/item/clothing/glasses/sunglasses/advanced/garb = 10,
+		//SPOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOL
+		/obj/item/reagent_containers/food/drinks/ftliver = 8,
+		/obj/item/megaphone = 7,
+		/obj/item/paicard = 5,
+	)
+
+//Master At Arms
+/datum/job/master_at_arms
+	mail_goodies = list(
+		/obj/item/ship_weapon/ammunition/naval_artillery/cannonball = 15,
+		/obj/machinery/nuclearbomb/beer = 10,
+		/obj/item/ship_weapon/ammunition/naval_artillery/armed = 10,
+		/obj/item/ship_weapon/parts/missile/warhead/plushtide = 10,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/torp_juice = 8,
+		//EARS RINGING, GUNS SINGING!!!
+		/obj/item/banner/munitions = 5,
+		//ITS CODED, WE SHOULD USE IT
+		/obj/item/ship_weapon/ammunition/naval_artillery/homing = 1,
+	)
+
+//Munitions Technician
+/datum/job/munitions_tech
+	mail_goodies = list(
+		/obj/item/powder_bag = 15,
+		/obj/item/radio/headset/munitions/munitions_tech/alt = 10,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/torp_juice = 8,
+		/obj/item/lighter = 5,
+		//Just like how the MAA used to make 'em!
+		/obj/item/slime_cookie/oil = 4,
+		/obj/structure/sign/poster/contraband/have_a_puff = 3,
+		/obj/item/powder_bag/plasma = 2,
+		/obj/item/clothing/head/ship/rising_sun = 1,
+		)
+
+//Pilot
+/datum/job/pilot
+	mail_goodies = list(
+		/obj/item/toy/crayon/spraycan = 15,
+		/obj/item/clothing/glasses/sunglasses = 10,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/torp_juice = 8,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/kamikaze = 8,
+		/obj/item/fighter_component/engine/flooded = 5,
```

Comment:
```suggestion
		/obj/item/fighter_component/engine/flooded = 5,
		/obj/item/key/fighter_tug = 3,
		/obj/item/clothing/head/ship/rising_sun = 2
```
In case they forget their keys, and obviously the pilots should get headbands as well!